source/partials/homepage/_contact.slim

```diff
@@ -23,4 +23,4 @@
         )
       .ContactsGrid-address
         p.Text.Text--light.Text--strong
-          ' Centro Empresarial de Braga
+          ' Pólo de Negócios de Braga
```

Comment:
Since we didn't have it before, I don't see the reasoning to have the door number now. Also, this was never the complete office address, I guess that will be solved in the upcoming redesign